project_proposal.Rmd

```diff
@@ -0,0 +1,23 @@
+---
+title: "project_proposal"
+author: "Neel Phaterpekar"
+date: "20/11/2020"
```

Comment:
I personally prefer the ISO YYYY-MM-DD format, since it removes any ambiguity and is a best practice for dates. 
```suggestion
date: "2020/11/20"
```